bgpd/bgp_io.c

```diff
@@ -401,6 +399,7 @@ static uint16_t bgp_write(struct peer *peer)
 		case BGP_MSG_UPDATE:
 			atomic_fetch_add_explicit(&peer->update_out, 1,
 						  memory_order_relaxed);
+			uo++;
```

Comment:
Why this parallel increment instead of simply using the already existing value (`peer->update_out`), as was the case prior to this change?

Reply:
this now parallels the update_last_write and how last_write is updated.  Either they both should use local variable or neither should.  The old code, in theory, could result in last_write being updated and last_update not, even in the BGP_MSG_UPDATE case.